spec/factories/person_defendants.rb

```diff
@@ -15,5 +15,21 @@
     arrestSummonsNumber { 'Random string' }
     employer_organisation { nil }
     employerPayrollReference { 'Random string' }
+
+    factory :realistic_person_defendant do
+      association :person, factory: :realistic_person
+      association :bail_status, factory: :realistic_bail_status
+      association :custodyTimeLimit, factory: :realistic_custody_time_limit
+      association :employer_organisation, factory: :realistic_organisation
+      bailConditions { Faker::Lorem.sentence }
+      bailReasons { Faker::Lorem.sentence }
+      perceivedBirthYear { Faker::Number.between(from: 1, to: 75) }
```

Comment:
I think this should be the year of birth rather than age - something like `Faker::Number.between(from: 1900, to: 2000)`

Reply:
Fixed by https://github.com/ministryofjustice/hmcts-common-platform-mock-api/pull/132